### PR TITLE
Improve pointers to README files and rename README.Dec

### PR DESCRIPTION
**Pull request by MatthewDaggitt**

### CODE REVIEW

File: src/Relation/Nullary.agda [hunk 1]
```diff
@@ -25,18 +26,22 @@ infix 2 _because_
...
--- value to compute independently from the proof portion.
--- This often allows good computational properties when we
--- only care about the boolean portion.
+------------------------------------------------------------------------
+-- Decidability.
+
+-- This definition allows the boolean portion of the value to compute
+-- independently from the proof portion. This translates to better
+-- computational properties when we only care about the result and not
+-- the proof. See README.Decidability for further details.
```
Comment (laMudri) [inline src/Relation/Nullary.agda]: Looking back, I think “the value” should be something more informative, like “a decision procedure”.

**Comment by laMudri** [comment]
It's a bit of a shame that these pointers are not clickable/go-to-definition-able (because of cyclic dependencies, I guess).

**Comment by gallais** [comment]
> It's a bit of a shame that these pointers are not clickable/go-to-definition-able (because of cyclic dependencies, I guess).

We could write a small Haskell script that spots special syntax in comments
and turns it into links. This way the html hosted by us would have the relevant
links.

A better solution would of course to extend Agda to do so natively.

File: src/Relation/Nullary.agda (hunk 1 above)
Comment (MatthewDaggitt) [reply src/Relation/Nullary.agda]: Updated the text.

**Pull request merged**